src/twisted/application/runner/_runner.py

```diff
@@ -118,15 +164,14 @@ def startLogging(self):
     def startReactor(self):
         """
         Register C{self.whenRunning} with the reactor so that it is called once
-        the reactor is running and start the reactor.
-        If L{RunnerOptions.reactor} is specified in C{self.options}, use that
-        reactor; otherwise use the default reactor.
+        the reactor is running, then start the reactor.
```

Comment:
This doesn't document an important side-effect; it _sets_ the reactor attribute.